frontend/wallet/src/render/DownloadLogic.re

```diff
@@ -109,3 +120,22 @@ let rec download = (filename, url, encoding, maxRedirects, chunkCb, doneCb) => {
     );
   Https.Request.onError(request, e => doneCb(Error(e)));
 };
+
+let codaRepo = "...";
```

Comment:
Still need to figure out where we wanna store the builds and if we wanna support linux + mac setup